### PR TITLE
Bug 1141406 - [TextSelection] Marionette, add method to get frequency of visibility changed

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1141406

### CODE REVIEW

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6ef9e6bf43c3)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=50ec57d6eddc)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=185b91e91b78)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=cc58f9dcc452)

File: apps/system/test/marionette/lib/text_selection.js [hunk 1]
```diff
@@ -29,6 +29,39 @@ TextSelection.prototype = {
 
   },
 
+  startCountVisibilityChanged: function() {
+    this.client.switchToFrame();
+    this.client.executeScript(function(query) {
+      var element = document.querySelector(query);
+      element.dataset.showSummary = 0;
+      var bubbleInitActive= element.classList.contains('active');
```
Comment (alivedise) [inline apps/system/test/marionette/lib/text_selection.js]: nit: space before =

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=516e22e8c7c8254a4023a443bd93c14a948c4659

**Pull request merged**